shell.nix

```diff
@@ -1,22 +1,26 @@
-let overlay = self: super:
```

Comment:
Formatted with `nixpkgs-fmt`, added `generate` and `testu01-stdin`, removed `libiconv` (don't know what that is).

Reply:
Nooo - I need libiconv on macos

Reply:
I'll add it back then!

Reply:
Done